travis-ci/unlock-stack.sh

```diff
@@ -4,8 +4,13 @@ set -e
 
 . ./travis-ci/set-env-vars.sh
 
-cd example || exit 1
-yarn
+(
+  set -e
 
-echo Unlocking stack
-node ./scripts/lock-stack.js false $DEPLOYMENT
\ No newline at end of file
+  cd example
+  rm -rf node_modules
+  npm install @cumulus/common
```

Comment:
~why are we installing @cumulus/common after destroying the package install?~
Ok, why are we nuking node_modules here before installing common, or is it that we've gotten rid of it already and need the common install to unlock the stack?

Reply:
I've pulled in the "use `npm ci` instead of `npm install`" changes into this PR, which makes this no longer necessary.  (`npm ci` will delete the node_modules directory on its own to ensure a clean install)

Reply:
I was running into issues where running `npm install @cumulus/common` after running `lernal bootstrap` was causing failures.

Reply:
Is this reproducible in a local Travis container?